runtime-scientific/ucx/autobuild/prepare

```diff
@@ -0,0 +1 @@
+sh "$SRCDIR"/autogen.sh
```

Comment:
Again, drop this file as this is handled automatically by Autobuild.

Reply:
I thought it was an AOSC convention. Sorry about it.